docs/source/conf.py

```diff
@@ -143,17 +145,29 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-# html_theme = 'default'
+html_theme = 'pydata_sphinx_theme'
```

Comment:
I'm confused, didn't we use that already?

Reply:
I've just moved it to the location suggested by the config file layout

Reply:
Ok.